CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v4.x.x
```

Comment:
Let's make it 4.4.0 (since there's a new feature)? And add a release date

Reply:
yeah, i've flagged it as a new feature but it was actually a bug fix. Updated changelog with version and release date